Service/src/routes/charts.py

```diff
@@ -96,8 +106,13 @@ def formatDataInterDay(jsonData):
 
 
 def formatDataIntraday(jsonData):
-   data = [] 
+   data = []
+   NoneItemsIEX = 0 
```

Comment:
poorly named variable -- you can do better naming here

with Python, there's no explicit types so you have to use variable names to communicate the type

If I understand this variable correctly it's `emptyIEXDataObjectCount`

Reply:
Not necessary anymore, so it is removed regardless

Reply:
why are the changes still here then?

Reply:
I deleted everything but the declaration, mistake